documentation/docs/mock-apps/verify-app.md

```diff
@@ -31,6 +31,15 @@ sequenceDiagram
 ```
 
 ## Rendering Verified Credential
+
 The UI of this page includes these information fields: Type, Issued by and Issue date. Besides, the page also contains the tab panel for HTML template, JSON data and the download button.
 
 With the download button, it will download the JSON data or JWT data if the credential is JWT.
+
+## Hash validation (if it exist in URL)
```

Comment:
Should we update this sentence to `Hash validation (if it exists in the URL)` to make it more grammatically correct?